libmatekbd/matekbd-keyboard-drawing.c

```diff
@@ -2206,6 +2206,7 @@ matekbd_keyboard_drawing_get_type (void)
 			sizeof (MatekbdKeyboardDrawing),
 			0,	/* n_preallocs */
 			(GInstanceInitFunc) matekbd_keyboard_drawing_init,
```

Comment:
@rbuj 

one question, the `','` in the line

`(GInstanceInitFunc) matekbd_keyboard_drawing_init,`

is needed?

Reply:
@sc0w Yes, it's a struct initialization and `(GInstanceInitFunc) matekbd_keyboard_drawing_init` is a function pointer.

Reply:
https://developer.gnome.org/gobject/stable/gobject-Type-Information.html#GTypeInfo
```C
struct GTypeInfo {
  /* interface types, classed types, instantiated types */
  guint16                class_size;

  GBaseInitFunc          base_init;
  GBaseFinalizeFunc      base_finalize;

  /* interface types, classed types, instantiated types */
  GClassInitFunc         class_init;
  GClassFinalizeFunc     class_finalize;
  gconstpointer          class_data;

  /* instantiated types */
  guint16                instance_size;
  guint16                n_preallocs;
  GInstanceInitFunc      instance_init;

  /* value handling */
  const GTypeValueTable *value_table;
};